packages/mobile/src/screens/chat-screen/ChatUserListItem.tsx

```diff
@@ -0,0 +1,165 @@
+import { useCallback } from 'react'
+
+import { chatActions, chatSelectors } from '@audius/common'
+import type { User } from '@audius/common'
+import { Text, View, TouchableOpacity } from 'react-native'
+import { useDispatch, useSelector } from 'react-redux'
+
+import IconUser from 'app/assets/images/iconUser.svg'
+import LoadingSpinner from 'app/components/loading-spinner'
+import { ProfilePicture } from 'app/components/user'
+import { UserBadges } from 'app/components/user-badges'
+import { makeStyles } from 'app/styles'
+import { useThemeColors } from 'app/utils/theme'
+
+const { createChat } = chatActions
+const { getCanMessageUser } = chatSelectors
+
+const messages = {
+  followsYou: 'Follows You',
+  followers: 'Followers',
+  cannotMessage: 'Cannot Be Messaged'
+}
+
+const useStyles = makeStyles(({ spacing, palette, typography }) => ({
+  rootContainer: {
+    backgroundColor: palette.white,
+    flexGrow: 1
+  },
+  profilePicture: {
+    height: spacing(18),
+    width: spacing(18)
+  },
+  border: {
+    borderBottomColor: palette.neutralLight4,
+    borderBottomWidth: 1
+  },
+  userContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    padding: spacing(4)
+  },
+  userNameContainer: {
+    flexGrow: 1,
+    display: 'flex',
+    flexDirection: 'column',
+    marginLeft: spacing(2.5),
+    gap: spacing(1)
+  },
+  userName: {
+    fontSize: typography.fontSize.small,
+    fontWeight: 'bold',
+    color: palette.neutral
+  },
+  followContainer: {
+    marginTop: spacing(1),
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-between'
+  },
+  handle: {
+    fontSize: typography.fontSize.small,
+    color: palette.neutral
+  },
+  followersContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center'
+  },
+  followersCount: {
+    fontWeight: '800',
```

Comment:
Do we have typography constants for weights?

Reply:
Ah man, just remembered why I did this - we do have constants but don't have one for 800. There is something wonky with our font weight constants not matching up with the specs in figma as well. Something that's on the to-do list.

Reply:
EDIT just gonna use `bold` here, confirmed with design that this is ok.